### PR TITLE
Hw1 Project Proposal and Contribution .md Files

### PR DESCRIPTION
**Pull request by ErickCif**

### CODE REVIEW

File: eac2277.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Open Source Development
+
+Name: Erick Cifuentes
+
+UNI: eac2277
+
+Github: [link](https://github.com/ErickCif)
+
+GitHub README: [link](https://github.com/ErickCif/ErickCif/blob/main/README.md)
```
Comment (timkpaine) [inline eac2277.md:9]: need an "extra" on readme like a badge or embeddable app.

File: eac2277.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+
+## Project Proposal
+[link](https://github.com/ErickCif/project-proposals-s2023/blob/HW1-Proposal-Music-Ping%2C-Next.js-Bug/Project/QueueR.md)
```
Comment (timkpaine) [inline eac2277.md:13]: should be relative link e.g. `[link](./projects/javascript/QueueR.md)`

File: eac2277.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+## Project Contributions
+[link](https://github.com/ErickCif/project-proposals-s2023/blob/HW1-Proposal-Music-Ping%2C-Next.js-Bug/Contribution/Next-jsBug.md)
```
Comment (timkpaine) [inline eac2277.md:16]: relative link

File: Contribution/Next-jsBug.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+## Next.js, I feel indebted.
```
Comment (timkpaine) [inline Contribution/Next-jsBug.md:1]: move to `projects/javascript/nextjs.md`

File: Project/QueueR.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+## Disclaimer
```
Comment (timkpaine) [inline Project/QueueR.md:1]: move to `projects/javascript/QueueR.md`

**Review by timkpaine — CHANGES_REQUESTED** [review]
remove `img.png`, remove `.idea` folder

File: Project/javascript/Next-jsBug.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+## Next.js, I feel indebted.
+For all three SWE internships I've done so far, I've heavily relied on Next.js as my go-to framework
```
Comment (timkpaine) [inline Project/javascript/Next-jsBug.md:2]: rename to `nextjs.md`, no need for "-Bug"

File: Project/javascript/Next-jsBug.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
+## Next.js, I feel indebted.
```
Comment (timkpaine) [inline Project/javascript/Next-jsBug.md:1]: should be `projects` not `Project`, lowercase plural

File: eac2277.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+
+UNI: eac2277
+
+Github: [link](https://github.com/ErickCif)
+
+GitHub README: [link](https://github.com/ErickCif/ErickCif/blob/main/README.md)
+
+
+## Project Proposal
+[link](https://github.com/ErickCif/project-proposals-s2023/blob/HW1-Proposal-Music-Ping%2C-Next.js-Bug/Project/javascript/QueueR.md)
```
Comment (timkpaine) [inline eac2277.md:13]: use relative link like `./projects/javascript/QueueR.md`

**Review by timkpaine — CHANGES_REQUESTED** [review]

**Comment by ErickCif** [comment]
Added relative links, fixed directories

**Pull request closed without merging**